tests/reference/run_dbg-test_raise_01-dfd86ca.stderr

```diff
@@ -1,7 +1 @@
-  File "tests/runtime_errors/test_raise_01.py", line 1
-    def test():
-  File "tests/runtime_errors/test_raise_01.py", line 4
-    test()
-  File "tests/runtime_errors/test_raise_01.py", line 2
-    raise
-ERROR STOP
+semantic error: The `runtime stacktrace` is not enabled. To get the stacktraces, re-build LPython with `-DWITH_RUNTIME_STACKTRACE=yes`
```

Comment:
You should build the `lpython` project with `-DWITH_RUNTIME_STACKTRACE=yes`. You can simply use the provided `build0.sh` and `build1.sh` files for building which has this options already setup.

Reply:
I did as you suggested and this error appeared, I guess I missed something while implementing the function but I can't figure out what exactly

![image](https://github.com/lcompilers/lpython/assets/131595768/96ee8c11-ef54-4e59-be18-61cc0c0265bc)

Reply:
@farah-salama the issue occurs because you have not currently implemented the method for compile-time strings. You can have a look at how `startswith` is implemented and add support for them.